docs/articles/instances.md

```diff
@@ -0,0 +1,41 @@
+# Instances
+
+## What is an Instance?
+Instances are the Marathon representation of [⇒Tasks](tasks.md) or [Task Groups](task-groups.md) in Mesos. An instance can exist even if there is currently no Task associated. We introduced this abstraction in order to support Mesos TaskGroups (called [⇒Pods](pods.md) in Marathon), where an instance can contain more than one Task. Beyond support for Pods, this abstraction also allows us to represent scheduled instances, which do not yet have Mesos Tasks associated with them. 
+
+### Instance Goals
+Depending on the goal of a given instance, Marathon will take actions to reach that goal: it might match offers in order to launch a Task or TaskGroup. In this case, these Tasks will eventually be associated with the instance. If an instance shall be stopped, Marathon will send kill requests to Mesos and eventually expunge the instance from its state.
+
+Currently, instance goals are only used internally, but are reflected in the API. Supported goals are
+* `Running` means the goal is to have a Mesos Task or TaskGroup associated with this instance, which should be running all the time.
+* `Stopped` means, all Mesos Tasks or TaskGroups associated with this instance shall be stopped. The instance and all its metadata will be retained. It is possible to change the goal to `Running` again.
+* `Decommissioned` means the Mesos Task or TaskGroup associated with this instance will be killed, and reservations and volumes will be destroyed. The instance cannot be revived, and once in a reportedly terminal state, it will be expunged from Marathon's state.
```

Comment:
I feel like there is value in adding a link to Goals.scala . https://github.com/mesosphere/marathon/blob/master/src/main/scala/mesosphere/marathon/core/instance/Goal.scala
and there is a `Terminal` goal

Reply:
The link makes sense – however `Terminal` is only a trait, not a Goal on its own.